frontend/src/locales/en.yml

```diff
@@ -1042,6 +1043,18 @@ sql-editor:
   no-rows-found: No rows found
   download-as-csv: Download as CSV
   download-as-json: Download as JSON
+  only-select-allowed: Only {select} statements are allowed to execute.
+  want-to-change-schema: If you want to {changeschema}.
+  change-schema: change schema
+  go-to-alter-schema: You can click the {alterschema} button, and submit an issue.
+  table-schema-placeholder: Select a table to see its schema
+  notify-empty-statement: Please input your SQL statements in the editor
+  notify-multiple-statements: >-
+    Detecting you have multiple SQL statements. SQL Editor only supports
```

Comment:
Multiple SQL statements detected. SQL Editor only executes the first statement. You can select another statement and execute it individually.